kubernetes/schema_container.go

```diff
@@ -517,6 +517,13 @@ func containerFields(isUpdatable, isInitContainer bool) map[string]*schema.Schem
 			Default:     "/dev/termination-log",
 			Description: "Optional: Path at which the file to which the container's termination message will be written is mounted into the container's filesystem. Message written is intended to be brief final status, such as an assertion failure message. Defaults to /dev/termination-log. Cannot be updated.",
 		},
+		"termination_message_policy": {
+			Type:         schema.TypeString,
+			Optional:     true,
+			ValidateFunc: validation.StringInSlice([]string{"File", "FallbackToLogsOnError"}, false),
+			Default:      "File",
```

Comment:
Adding new defaults will probably trigger a diff for existing configuration and people usually complain about that in non major releases. Maybe this should rather be computed.

Reply:
Just let me know what you want.... computed seems fine as the `File` default will be specified by k8s I believe, allowing us to override to `FallbackToLogsOnError`.

Reply:
@pdecat anyone get back to you? Any specific person we could contact here?

Reply:
👋 I agree with @pdecat that we should make this a `Computed` field in this case. Otherwise this PR looks good!